src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
Read about best practices of naming enums
```suggestion
public enum Color {

```